packages/manager/apps/web/client/app/domain/webhosting/order/steps/hosting/domain-webhosting-order-steps-hosting.constants.js

```diff
@@ -0,0 +1,16 @@
+export const OPTIONS_DETAILS = [
```

Comment:
Could we avoid it and do it using the API ?

Reply:
We can avoid it and so not changing the API, since the offers are display by order of pricing. Making the changes